jest.config.js

```diff
@@ -152,7 +160,15 @@ module.exports = {
     '/__tests__/__util__',
     '/webpack.dev.config.js',
     '/src/server/server-test.js',
-    '/src/server/server.config.js'
+    '/src/server/server.config.js',
+    'src/server/blockchain',
+    'src/server/db',
+    'src/server/gun',
+    'src/server/loadtest',
+    'src/server/login',
+    'src/server/mautic',
+    'src/server/send',
+    'src/server/storage',
```

Comment:
what is all these?

Reply:
done. removed it. I added this to run specific tests locally. And pushed it by mistake.